doc/api/events.md

```diff
@@ -208,7 +231,19 @@ ee2[Symbol.for('nodejs.rejection')] = console.log;
 Setting `events.captureRejections = true` will change the default for all
 new instances of `EventEmitter`.
 
-```js
+```mjs
+import * as events from 'node:events';
+
+events.captureRejections = true;
```

Comment:
@jasnell After change `import events from 'node:events'` to `import * as events from 'node:events'`, the `events.captureRejections` assign will fail. lint is not happy, it says `The members of 'events' are read-only` . 

Change `events.captureRejections` to  `EventEmitter.captureRejections` will works, but the doc recommend using `events.captureRejections` instead of `EventEmitter.captureRejections`   to change the default. 

I am thinking the two paths:
1. change `import * as events from 'node:events';` back to `import events from 'node:events';`
2. recommend using `EventEmitter.captureRejections` to change default behavior: 
    * replace all `events.captureRejections` to `EventEmitter.captureRejections`
    * add a `captureRejections` static field(`EventEmitter.captureRejections`] entry to the [`EventEmitter` class part](https://nodejs.org/api/events.html#class-eventemitter)
    * maybe we should remove [the `events.captureRejections` paragraph ](https://nodejs.org/api/events.html#eventscapturerejections) to reduce confusion?

What do you suggest to fix this problem?

Reply:
For that, changing to `import { EventEmitter } from 'node:events'` and using `EventEmitter.captureRejections = true` is likely the way to go.

Reply:
Fixed, PTAL, thanks.